test/test-sock-accept.c

```diff
@@ -0,0 +1,156 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include "uv.h"
+#include "uvwasi.h"
+#include "wasi_serdes.h"
+
+#define PREOPEN_SOCK 3
+#define INVALID_SOCK 42
+
+#define CONNECT_ADDRESS "127.0.0.1"
+#define TEST_PORT_1 10500
+#define CONNECTION_WAIT_TIME 2000
+int delayedThreadTime = 4000;
+int immediateThreadTime = 0;
```

Comment:
Please use snake case instead of camelCase.

Reply:
Pushed commit to fix that.